cpp/upgrades/bd182f697bf1316c401421d64e582871331a69f4/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Add some unions to the dbscheme
+compatibility: full
```

Comment:
I don't think this is full compatibility (see the definitions in https://wiki.semmle.com/display/IN/Creating+an+ODASA+Database+Upgrade+Script), unless we provide an upgrade script that fills out the new relations (which seems straightforward, if a bit tedious).

Reply:
It seems like the C# commit doing the same kind of change claimed full compatibility in https://git.semmle.com/Semmle/code/pull/9836, though? Or am I mistaken?

Reply:
Oh I see what's going on now - the new things in the dbscheme are all unions (of the form `@a = @b | @c`), the primitive things they are made from are all there in the snapshot already.  I should have spotted that there are no extractor changes either, because nothing's changing in the snapshot.

My mistake.